composable_kernel/include/utility/static_buffer_of_vector_type_v2.hpp

```diff
@@ -104,6 +104,11 @@ struct StaticBufferOfVectorTypeV2 : public StaticallyIndexedArray<T, N>
             [&](auto i) { GetElement(i, true) = invalid_element_value_; });
     }
 
+    __host__ __device__ void Fill(VecBaseType v)
```

Comment:
StaticBufferOfVectorTypeV2 should be deprecated. StaticBufferTupleOfVector should be used instead

Reply:
Done

Reply:
Done